### ISSUE TITLE
MacOS: ftxvalidator is installed, but "Checking with ftxvalidator" still returns ERROR

### ISSUE THREAD

**Issue by eliheuer** [googlefonts/fontbakery#1936]
On `MacOS 10.11` I have `ftxvalidator [build 207, FontToolbox.framework build 257]` installed and have tested it from the command line on a `.ttf` file, this results in a plist as output.

However, when I run `fontbakery check-googlefonts Same-Font.ttf` I still get a checking with `ftxvalidator` ERROR:

```
>> com.google.fonts/check/035 with (('font[0]', 'Orbitron-VF.ttf'),)
   Checking with ftxvalidator.
 * ERROR: Failed with AttributeError: module 'plistlib' has no attribute 'readPlistFromString'
           File "/Users/eli/Py/Venv/type-tools/lib/python3.6/site-packages/fontbakery/checkrunner.py", line 347, in _exec_check_generator
             for sub_result in gen:
         

   Result: ERROR
```

**Comment by felipesanches** [comment]
Looking here, `readPlistFromString` should be the correct method:
https://docs.python.org/2/library/plistlib.html

But then looking at the python 3 documentation:
https://docs.python.org/3/library/plistlib.html

we see that we have `plistlib.readPlistFromBytes(data)` instead!

**Comment by eliheuer** [comment]
I just made the above change and it worked, thanks! 

If anyone else has this problem the file that needs to be updated is here: `../Lib/fontbakery/specifications/general.py`

Could we try doing a try/except block here to avoid this error?

**Comment by felipesanches** [comment]
yes, I think that would work nicely.

**Comment by eliheuer** [comment]
Ok, I'll try making a pull request soon.

**Comment by felipesanches** [comment]
this was fixed on pull request #1947
